tumblr/kanvas-ios
# Use selected font for the font selector button

Adds `fontSelectorUsesFont` setting to use the currently selected font for the font selector button's label.

## Testing

* Enable the `fontSelectorUsesFont` setting ("Editor uses font for font selector" in the example app)
* Add text
* Tap the text button to change the font
* Notice the label font change as the font changes

https://user-images.githubusercontent.com/3250/104977234-45850000-59bc-11eb-9843-1e875d5dba21.mov
